values.yaml

```
@@ -387,3 +387,116 @@ connectInject:
     # configured proxy.
     proxyDefaults: |
       {}
+
+# Mesh Gateways enable Consul Connect to work across Consul datacenters.
+meshGateway:
+  # If mesh gateways are enabled, a Deployment will be created that runs
+  # gateways and Consul Connect will be configured to use gateways.
+  # See {todo doc link} for more information.
+  # Requirements: consul >= 1.6.0 and consul-k8s >= 0.8.2 if using global.bootstrapACLs.
+  enabled: false
+
+  # Which mode the gateway should run in. Can be either 'remote' or 'local'.
+  # If set to remote, Connect traffic to services in other datacenters will
+  # be sent directly to the other datacenter's gateway.
+  # If set to local, Connect traffic to services in other datacenters will
+  # be proxied via the local gateway. Use 'local' if services cannot directly
+  # connect to the other datacenter's gateway and so must proxy through the local
+  # gateway, i.e. if only the gateways can communicate between datacenters.
+  mode: remote
+
+  # Number of replicas for the Deployment.
+  replicas: 2
+
+  # What gets registered as wan address for the gateway.
+  wanAddress:
+    # Port that gets registered.
+    port: 443
+
+    # If true, each Gateway Pod will advertise its NodeIP
+    # (as provided by the Kubernetes downward API) as the wan address.
+    # This is useful if the node IPs are routable from other DCs.
+    # The port key will be used as the port for the wan address.
```

What do you mean by `port key`? Something like this? 

```suggestion
    # The port configured above will be used as the port for the wan address.
```

We use `key` elsewhere in this file in several different ways, so overloading it here as well will get confusing.

---

✅